hazelcast/src/test/java/com/hazelcast/wan/impl/WanNoDelayReplicationTest.java

```diff
@@ -0,0 +1,57 @@
+package com.hazelcast.wan.impl;
+
+import com.hazelcast.map.impl.wan.MapReplicationUpdate;
+import com.hazelcast.test.AssertTask;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.wan.ReplicationEventObject;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertSame;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.spy;
+
+/**
+ * @author Henri Tremblay
+ */
```

Comment:
FYI: Nightly tests don't count for the code coverage measurement and are not executed in the PR builder. So if these tests are important for one of those two things, we should keep them as separate quick (PR builder + coverage) or slow test (coverage).